releases/release-5.0.6.md

```diff
@@ -0,0 +1,168 @@
+---
+title: TiDB 5.0.6 Release Notes
+category: Releases
+---
+
+# TiDB 5.0.6 Release Notes
+
+Release date: December 28, 2021
+
+TiDB version: 5.0.6
+
+## Compatibility changes
+
++ Tools
+
+    + TiCDC
+
+        - Output the cdc server cmd error from stdout to stderr. [#3133](https://github.com/pingcap/tiflow/issues/3133)
+
+## Improvements
+
++ TiDB
+
+    (dup) - Fix the issue that optimistic transaction conflicts might cause transactions to block each other [#11148](https://github.com/tikv/tikv/issues/11148)
+    (dup) - Fix the issue of false positive error log `invalid cop task execution summaries length` for MPP queries [#1791](https://github.com/pingcap/tics/issues/1791)
+    (dup) - Show the affected SQL statements in the debug log when the coprocessor encounters a lock, which is helpful in diagnosing problems [#27718](https://github.com/pingcap/tidb/issues/27718)
+
++ TiKV
+
+    - Increase the speed of inserting SST files by moving the verification process to import thread pool from apply thread pool [#11239](https://github.com/tikv/tikv/issues/11239)
+    - Add metrics for the garbage collection module of Raft logs to locate performance problems in this module [#11374](https://github.com/tikv/tikv/issues/11374)
+    - Collapse some uncommon metrics related to storage in Grafana dashboard [#11681](https://github.com/tikv/tikv/issues/11681)
+
++ PD
+
+    (dup) - Speed up the exit process of schedulers [#4146](https://github.com/tikv/pd/issues/4146)
+    - Make the `scatter-range-scheduler` scheduler work better by allowing to schedule empty regions and fix configurations [#4497](https://github.com/tikv/pd/issues/4497)
+
++ Tools
+
+    + TiCDC
+        (dup) - Optimize rate limiting control on TiKV reloads to reduce gPRC congestion during changefeed initialization [#3110](https://github.com/pingcap/ticdc/issues/3110)
+        (dup) - Fix the issue that scanning stock data might fail due to TiKV performing GC when scanning stock data takes too long [#2470](https://github.com/pingcap/tiflow/issues/2470)
+        (dup) - Fix the issue that changefeed does not fail fast enough when the ErrGCTTLExceeded error occurs [#3111](https://github.com/pingcap/ticdc/issues/3111)
+        (dup) - Add a tick frequency limit to EtcdWorker to prevent frequent etcd writes from affecting PD services [#3112](https://github.com/pingcap/ticdc/issues/3112)
+        - Support batch messages to reduce EtcdWorker tick. [3112](https://github.com/pingcap/tiflow/issues/3112)
```

Comment:
```suggestion
        - Add a tick frequency limit to EtcdWorker to prevent frequent etcd writes from affecting PD services [3112](https://github.com/pingcap/tiflow/issues/3112)
```

Reply:
感觉中英文的侧重不太一样？`支持消息 Batch 操作以减少 EtcdWorker tick 频率`

Reply:
@overvenus 请看下用哪个表达比较合适？

Reply:
麻烦用 中文 的表达吧